packages/gatsby-source-nacelle/README.md

```diff
@@ -0,0 +1,91 @@
+<p align="center">
+  <a href="https://www.getnacelle.com">
+    <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
+    <img alt="Nacelle" src="https://raw.githubusercontent.com/getnacelle/gatsby-theme-nacelle/master/examples/store-with-checkout/src/images/nacelle-rocket-icon-crop.png" width="60" />
+  </a>
+</p>
+
+<h1 align="center">
+  Gatsby-Source-Nacelle
+</h1>
+
+<!-- [![npm version](https://img.shields.io/npm/v/@nacelle/gatsby-source-nacelle.svg)](https://www.npmjs.com/package/@nacelle/gatsby-source-nacelle) -->
+<!-- [![GitHub license](https://img.shields.io/github/license/getnacelle/nacelle-react/tree/master/packages/gatsby-source-nacelle)](https://github.com/getnacelle/nacelle-react/tree/master/packages/gatsby-source-nacelle/blob/master/LICENSE) -->
+
+This plugin connects Gatsby to [Nacelle's](https://www.getnacelle.com) Hail Frequency API, which gives you access to the product data (individual products, collections, etc.) and content data (blog posts, articles, etc.) needed to build an eCommerce storefront.
+
+## What is Nacelle?
```

Comment:
This seems repetitive since it's also in the example? I think this info is good in packages, but probably less useful in the examples READMEs. When someone is looking at one of the examples, they probably already know what Nacelle is, right?

Reply:
I would hope so, but am not sure. In any event, I'm happy to remove it from this README.